container/jaas-login-module/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -1,11 +1,9 @@
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
We can revert the changes tho this file as well - was done while debugging JAAS, but are not needed.